db/column_family.cc

```diff
@@ -616,7 +619,7 @@ void ColumnFamilyData::RecalculateWriteStallConditions(
                imm()->NumNotFlushed() >=
                    mutable_cf_options.max_write_buffer_number - 1) {
       write_controller_token_ =
-          SetupDelay(ioptions_.delayed_write_rate, write_controller,
+          SetupDelay(mutable_db_options_.delayed_write_rate, write_controller,
```

Comment:
Oh, one more thing, what if in the future, if more options needs be to dynamically set, the `ColumnFamilyData` needs to use these options, by storing `mutable_db_options_` in `ColumnFamilyData` saves us more trouble. Because the `ColumnFamilyData` already has a member called `mutable_cf_options_`, probably we can store a `mutable_db_options_`?